src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -299,13 +299,20 @@ private async Task<IWebHost> BuildWebHost(ScriptApplicationHostOptions hostOptio
                 EnableDotNetWorkerStartup();
             }
 
+            EnableWorkerIndexing();
+
             return settings;
         }
 
         private void EnableDotNetWorkerStartup()
         {
             Environment.SetEnvironmentVariable("DOTNET_STARTUP_HOOKS", "Microsoft.Azure.Functions.Worker.Core");
         }
+        
+        private void EnableWorkerIndexing()
+        {
+            Environment.SetEnvironmentVariable("FunctionsHostingConfig__WORKER_INDEXING_ENABLED", 1.ToString());
```

Comment:
Can we do this after handling local.settings.json and only if the env var isn't defined already? It would be nice to leave customers a way to turn off worker indexing if necessary, and AFAIK setting this to 0 would be the best way for them

Reply:
@ejizba Not setting the env variable anymore if value is set in local.settings.json.